api/src/services/sample-location-service.ts

```diff
@@ -64,13 +64,24 @@ export class SampleLocationService extends DBService {
    */
   async insertSampleLocations(sampleLocations: PostSampleLocations): Promise<SampleLocationRecord[]> {
     const methodService = new SampleMethodService(this.connection);
-
+    const determineName = (geometry: Feature<Geometry | GeometryCollection, Properties>) => {
```

Comment:
Nit: Add return type and JSdoc (or add it to the parent method) explaining how the name is determined)